### ISSUE TITLE
Documentation is flaky on units

### ISSUE THREAD

**Issue by thoni56** [libharu/libharu#150]
There are no specification of the units that are returned or used. E.g. ` HPDF_Font_TextWidth` in what unit is the width returned? Which unit of font size is `HPDF_Page_SetFontAndSize()` expecting?

This is a common issue with many of the API definitions in the documentation. It would also need a short section the relation between points, pixels, font sizes, and whatever other units are used.

**Comment by fg123** [comment]
Agreed, the documentation definitely needs a bit of work. I answered your question on Stack Overflow so I'm linking that answer to here as well: https://stackoverflow.com/questions/42320887/hpdf-units-for-text-width-and-height/46508890#46508890

**Comment by chakani** [comment]
Pixels are not relevant, PDF only works with points (72 points/inch). If you are converting from a pixel-based context (eg. Windows GDI), you need to declare an arbitrary pixels/inch, and convert all your pixels based on this value.
